schemas/netconf_managers.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "urn:nuage-metroae:netconfmanager",
+    "title": "Netconf Manager",
```

Comment:
"NETCONF Managers". Note the upper case and the plural. This schema contains a list...